tests/integration/test_migrator.py

```diff
@@ -142,7 +164,41 @@ def test_continuous_migration_handles_multiple_records(app, record_1502655_and_1
     assert expected == result
 
 
+def test_continuous_migration_avoids_jamming(app, many_times_same_record):
+    with app.app_context():
```

Comment:
You already have an application context: https://github.com/inspirehep/inspire-next/blob/319648c3d30afdbe2b34672beac6027b8858aa94/tests/integration/conftest.py#L50